packages/core/template/default/404.md

```diff
@@ -1,7 +1,7 @@
 <frontmatter>
-    title: Page not found
-    layout: 404.md
+  title: Page not found
+  layout: 404.md
 </frontmatter>
 
 -><p style="font-size: 10rem">404</p><-
--><p style="font-size: 1.5rem">File not found<br>Click <a href="/">here</a> to go back to the home page.</p><-
```

Comment:
I think this is fine. Is there any reason that you changing it? I'm guessing the reason why @EltonGohJH initially have this wording is because it mirrors the existing [github pages 404 page](https://httpwg.org/specs/hjkhk.image)

Reply:
I see. Initially I changed the wording for "file not found" to "page not found" (both seem to be widely used for 404 pages) as the wording felt slightly awkward to me (since the deployed MarkBind site would be dealing with pages). If we want to mirror the existing github pages, then we can keep the "file not found" but add in the "The site configured at this address does not contain the requested file. If this is your site, make sure that the filename case matches the URL." Does this sound alright?

Reply:
Just to clarify, I'm not suggesting we should copy the content from GitHub page 404 wholesale. I agree with the part about "file not found" could be awkward for pages that do not exist. On the other hand, this 404 also covers legitimate file not founds, i.e. if you go to https://markbind.org/image.png, this 404 is going to show up as well. If so, I think mentioning "file not found" is perfectly valid. In fact, now "page not found" is mentioned in the title, and "file not found" is mentioned in the content. I think this somewhat ensures that we covered all angles 🤣

Given that this is a template that is general enough at the moment, we can keep it as is. Users can make further modifications to customize the style and phrasing of their site.

What do you think?

Reply:
Agreed! And fixed in [4ebf484](https://github.com/MarkBind/markbind/pull/2165/commits/4ebf4845b49cb128b3f2812be191fd2f508c8c87) :smile: